components/collapsible/showmore/src/index.js

```diff
@@ -0,0 +1,52 @@
+import React, {useState} from 'react'
+import PropTypes from 'prop-types'
+import Button from '@s-ui/react-atom-button'
+import cx from 'classnames'
+
+const CollapsibleShowmore = ({
+  children,
+  copy = 'Show more',
+  maxHeight = 100
+}) => {
+  const baseClass = 'sui-CollapsibleShowmore'
+  const [heightContainer, setHeightContainer] = useState(maxHeight)
+  const [hideButton, setHideButton] = useState(false)
+
+  const buttonClassName = cx(`${baseClass}-actionButton`, {
+    'is-hidden': hideButton
+  })
+
+  const unfold = () => {
+    setHeightContainer('auto')
+    setHideButton(true)
+  }
+
+  return (
+    <div className={`${baseClass}`}>
+      <div
+        className={`${baseClass}-container`}
+        style={{height: heightContainer}}
```

Comment:
just so this doesn't grow, what about creating a variable? IT would then improve the readability of the render method. (but it's a totally personal opinion)